scripts/core/editor3/reducers/editor3.tsx

```diff
@@ -262,6 +262,12 @@ function escapeRegExp(string) {
     return string.replace(/[.*+?^${}()|[\]\\]/g, '\\$&');
 }
 
+function insertTab(editorState: EditorState, selection: SelectionState, tabCharacter: string) {
+    const newContent = Modifier.replaceText(editorState.getCurrentContent(), selection, tabCharacter);
```

Comment:
why not `Modifier.insertText`?

Reply:
ha... That's what I meant to put. I guess it was working anyway cause the selection is collapsed

Reply:
Actually, now that I think about it, a user could select some text, press TAB, and expect the text to be replaced by the tab character... I have some questions for Karel so I'll add this one to the list

Reply:
good point, `replaceText` might be useful in that case